### PR TITLE
Add set up details to development guidelines

### PR DESCRIPTION
**Pull request by MarahGamdou**
Added set up details to development guidelines to take into account possible set up issues

### CODE REVIEW

File: docs/development-guidelines.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ You need to have:
 
 - [rootless Docker](https://docs.docker.com/engine/security/rootless/)
   - Note that if Docker is running as root on you local machine, you may run into some permission issues. 
+  - Note that if you use Rootless mode, you might want to check that your Docker daemon is running properly, e.g., with `$ sudo systemctl status docker`. If your Docker daemon is inactive, you can start it manually using, e.g., with `$ sudo systemctl start docker`.
```
Comment (rbehjati) [inline docs/development-guidelines.md:9]: Is `sudo` necessary?

Comment (rbehjati) [inline docs/development-guidelines.md:9]: super-nit: The formatting script that we currently have does not automatically add line-breaks. Until we fix that, we have to add line-breaks manually. Can you add line breaks the new lines? :)

**Review by rbehjati — APPROVED** [review]
Nice. Thanks :)

File: docs/development-guidelines.md (hunk 1 above)
Comment (MarahGamdou) [reply docs/development-guidelines.md:9]: Not for `$ sudo systemctl status docker`. I will remove it.

**Pull request merged**